### PR TITLE
docs: add data-collect

### PR DESCRIPTION
**Pull request by hi-rustin**
Signed-off-by: hi-rustin <rustin.liu@gmail.com>

### CODE REVIEW

**Comment by hi-rustin** [comment]
/assgin @Mini256 

PTAL~ Thanks!

File: src/introductions/data-collect.md [hunk 1]
```diff
@@ -1 +1,44 @@
 # 数据收集方式
+
+## GH Archive
+
+### 收集方式
+
+devstats 会一次性将 [GH Archive] 所有的 JSON 文件逐个下载下来进行整合然后存储到 PostgreSQL 中，并且在该过程中会将数据分为固定数据和可变数据，并且用 event_id 作为可变数据的主键来进行维护和更新。
```
Comment (Mini256) [inline src/introductions/data-collect.md:7]:
```suggestion
devstats 会一次性将 [GH Archive] 所有的 JSON 文件逐个下载下来进行解析然后存储到 PostgreSQL 中，并且在该过程中会将数据分为固定数据和可变数据，将 event_id 作为可变数据的主键来进行维护和更新。
```

File: src/introductions/data-collect.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+
+### 收集工具
+
+devstats 使用 [gha2db](https://github.com/cncf/devstatscode/blob/master/cmd/gha2db/gha2db.go) 工具来分析和解析数据，并将数据存储到 PostgreSQL 中。
```
Comment (Mini256) [inline src/introductions/data-collect.md:11]:
```suggestion
devstats 使用 [gha2db](https://github.com/cncf/devstatscode/blob/master/cmd/gha2db/gha2db.go) 工具来解析数据，并将数据存储到 PostgreSQL 中。
```

File: src/introductions/data-collect.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+
+### 收集工具
+
+devstats 使用 [get_repos](https://github.com/cncf/devstatscode/blob/master/cmd/get_repos/get_repos.go) 来 clone/pull 仓库，并对仓库的 git log 逐一分析。
+
+## CNCF gitdm
+
+### 收集方式
+
+[CNCF gitdm] 提供了一个所有公司信息的 [JSON 文件](https://github.com/cncf/devstats/blob/master/github_users.json)，是一个 [Git LFS](https://git-lfs.github.com/)，devstats 会将所有的数据通过该 JSON 文件直接导入。devstats 会从中解析 GitHub 信息并且与其他数据结合，这样我们就收集到了大部分贡献者的公司信息。
```
Comment (Mini256) [inline src/introductions/data-collect.md:37]:
```suggestion
[CNCF gitdm] 提供了一个存储开发者所属公司信息的 [JSON 文件](https://github.com/cncf/devstats/blob/master/github_users.json)，该文件使用 [Git LFS](https://git-lfs.github.com/) 方式存储。devstats 会从中解析出 GitHub 用户的公司信息，并与其他数据结合，这样我们就收集到了大部分贡献者的公司信息。
```

**Pull request merged**